6-Final-Proj/server/tests/delete_todos.test.ts

```diff
@@ -0,0 +1,52 @@
+import { Testkit } from "./testkit";
+import { AxiosResponse, AxiosError } from "axios";
+import axios from "axios";
+import { TodoItem } from "../../common/types";
+import { getRandomItem, getRandomUserId } from "./utils";
+
+const testkit = new Testkit();
+
+describe("DELETE /todos", () => {
+  testkit.beforeAndAfterEach();
+
+  it("When the item exists, \
+  it should return 200 and the item should be deleted", async () => {
+    const userId = getRandomUserId();
+    const todo1: TodoItem = getRandomItem();
```

Comment:
instead of `getRandomItem` aRandomItem(item: Partial<TodoItem>)`. the impl should be a merge between random values (use chance.js) to the specific values that u passed.